server/src/main/java/org/elasticsearch/index/mapper/IgnoredFieldMapper.java

```diff
@@ -33,40 +40,63 @@
  * A field mapper that records fields that have been ignored because they were malformed.
  */
 public final class IgnoredFieldMapper extends MetadataFieldMapper {
-
+    public static final Version DOC_VALUES_VERSION = Version.V_8_0_0;
     public static final String NAME = "_ignored";
-
     public static final String CONTENT_TYPE = "_ignored";
 
     public static class Defaults {
         public static final String NAME = IgnoredFieldMapper.NAME;
 
         public static final FieldType FIELD_TYPE = new FieldType();
+        public static final FieldType LEGACY_FIELD_TYPE = new FieldType();
 
         static {
             FIELD_TYPE.setIndexOptions(IndexOptions.DOCS);
             FIELD_TYPE.setTokenized(false);
             FIELD_TYPE.setStored(true);
```

Comment:
It should be the other way around:
```suggestion
            FIELD_TYPE.setStored(false);
```